CMakeLists.txt

```diff
@@ -103,10 +103,14 @@ if ("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
   option(PARQUET_RPATH_ORIGIN
     "Build Parquet libraries with RATH set to \$ORIGIN"
     OFF)
+  option(PARQUET_MINIMAL_DEPENDENCY
+    "Depend only on Thirdparty headers to build libparquet. Always OFF if building binaries"
+    OFF)
```

Comment:
We could also implicitly set this value to ``OFF`` by default and set it to `ON` if `PARQUET_BUILD_TESTS` and `PARQUET_BUILD_EXECUTABLES` and `PARQUET_BUILD_BENCHMARKS` are all `OFF`